src/app/components/planner-list/planner-list.component.less

```diff
@@ -139,3 +203,100 @@
     align-items: inherit;
   }
 }
+
+button.btn.btn-link.dropdown-toggle:focus {
+  outline: none;
+}
+
+.planner-hack-title-truncate {
+  max-width: ~"calc(e('100% - 20px'))";
+}
+
+/// ng x-datatables css
+
+
+.cell-overflow {
+  overflow: visible !important;
+}
+
+.header-cell{
+  font-weight: bold !important;
+  overflow: visible !important;
+  color: #363636 !important;
+}
+
+.datatable-row-detail {
+  overflow-y: visible;
+}
+
+.datatable-header {
+  overflow: visible !important;
+}
+
+.fullscreen {
+  position: absolute !important;
+  height: auto !important;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+}
+
+a.quick-add-icon, .f8-action-icon {
+  opacity: 0;
+  text-decoration: none;
+}
+
+.ngx-datatable.material.single-selection .datatable-body-row.active,
+.ngx-datatable.material.single-selection .datatable-body-row.active .datatable-row-group,{
+    background-color: @color-pf-black-300 !important;
+}
+
+.ngx-datatable.material.single-selection .datatable-body-row.active:hover,
+.ngx-datatable.material.single-selection .datatable-body-row.active:hover .datatable-row-group, {
+    background-color: @color-pf-black-300 !important;
+}
+
+.ngx-datatable.material.single-selection .datatable-body-row.active:focus,
+.ngx-datatable.material.single-selection .datatable-body-row.active:focus .datatable-row-group{
+    background-color: @color-pf-black-300 !important;
+}
+
+.datatable-body-row:hover .wi-detail-icon, .datatable-body-row.active .wi-detail-icon {
+  opacity: 1;
+}
+.datatable-body-row:hover .f8-action-icon, .datatable-body-row.active .f8-action-icon {
```

Comment:
this css property is common. We can add classes directly at first instance rather than adding css property again